### DISCUSSION TITLE
amamba/user-guide/pipeline/config/agent

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4111] [Announcements]
# amamba/user-guide/pipeline/config/agent

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/pipeline/config/agent.html

**Comment by w-mf** [comment]
自定义镜像的倒是给个说明呀。这个示例也换行也不给注释说明，看不懂呀

**Comment by windsonsea** [comment]
already fixe, thanks.
